Package.swift

```diff
@@ -23,7 +26,12 @@ let package = Package(
     targets: [
         .target(
             name: "KlaviyoSwift",
-            dependencies: [.product(name: "AnyCodable", package: "AnyCodable")]),
+            dependencies: [.product(name: "AnyCodable", package: "AnyCodable")],
+            path: "Sources/KlaviyoSwift"),
```

Comment:
Actually what's this about?

Reply:
the path?

Reply:
yes.

Reply:
```
///   - path: The custom path for the target. By default, the Swift Package Manager requires a target's sources to reside at predefined search paths;
///       for example, `[PackageRoot]/Sources/[TargetName]`.
///       Don't escape the package root; for example, values like `../Foo` or `/Foo` are invalid.
```

I think if the name of the target is the same as the name of the directory under `/Source` we don't need this but now that we have two targets and products I was just being more explicit about the path of the source files.

Reply:
https://docs.swift.org/package-manager/PackageDescription/PackageDescription.html#product